### PR TITLE
 4 commits 

### PR DESCRIPTION
**Pull request by sergiomb2**
Let me now if you want that I do several pull requests.

Commit 1 : copied from https://github.com/projectM-visualizer/projectm/pull/173
Commit 2 : projectM-jack don't need hard coded -lSDL I use SDL2 so we don't need to link with SDL1 and we can build withou SDL-devel files
Commit 3: /usr/bin/projectM_jack => /usr/bin/projectM-jack
Commit 4: Remove unneeded files, they are regenerated with autotools

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -164,6 +164,11 @@ AX_CHECK_COMPILE_FLAG([-std=c++11], [
 AC_ARG_ENABLE([qt], AS_HELP_STRING([--enable-qt], [Enable Qt: needed for pulseaudio and jack GUIs]), [], [enable_qt=check])
 AS_IF([test "$enable_qt" != "no"],
   [
+    case $host_os in
+      linux*)
+        PATH="$PATH:`pkg-config --variable=host_bins Qt5Core`"
```
Comment (revmischa) [inline configure.ac]: comment this please... i have no idea what this is doing

File: src/projectM-jack/Makefile.am [hunk 2]
```diff
@@ -22,14 +22,13 @@ ${QT_CXXFLAGS} \
 
 # TODO: build qprojectM-jack and projectM-jack as separate programs
 
-bin_PROGRAMS = projectM_jack
+bin_PROGRAMS = projectM-jack
```
Comment (revmischa) [inline src/projectM-jack/Makefile.am]: this is okay i just hope it doesn't trip up downstream maintainers

File: configure.ac (hunk 1 above)
Comment (sergiomb2) [reply configure.ac]: hello , I'm studying this [1], while the work it is and hack ! 
I will try improve it today 
[1]
https://autotools.io/pkgconfig/pkg_check_modules.html

File: src/projectM-jack/Makefile.am (hunk 2 above)
Comment (sergiomb2) [reply src/projectM-jack/Makefile.am]: I forgot to mention this also is align with desktop file configuration , which try run projectM-jack

**Comment by sergiomb2** [comment]
I dropped Qt patch (Commit 1), it will be to review later in other PR

Commit 2 : projectM-jack don't need hard coded -lSDL I use SDL2 so we don't need to link with SDL1 and we can build withou SDL-devel files
Commit 3: /usr/bin/projectM_jack => /usr/bin/projectM-jack  also align with desktop file configuration , which try run projectM-jack
Commit 4: Remove unneeded files, they are regenerated with autotools

**Pull request merged**